data/roberanegussie.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Robera Negussie ",
+  "type": "personal",
+  "bio": "Software Developer **OSS enthusiast**.",
+  "links": [
+    {
+      "name": "My Twitter Handle",
+      "url": "https://twitter.com/roberanegussie",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "My LinkedIn Profile",
+      "url": "https://www.linkedin.com/in/roberatoye/",
+      "icon": "FaLinkedIn"
```

Comment:
Please replace it with **' FaLinkedin '**  :+1: 
You can also checkout [ICONS](https://linkfree.eddiehub.io/icons) for valid icons.
```Suggestion
      "icon": "FaLinkedin"
```